test/provisioner/provision_ibmcloud_kustomize.go

```diff
@@ -28,13 +28,9 @@ type IBMCloudInstallOverlay struct {
 
 type QuayTagsResponse struct {
 	Tags []struct {
-		Name       string `json:"name"`
-		ModifiedAt string `json:"last_modified"`
-		Digest     string `json:"manifest_digest"`
-		Size       string `json:"size"`
-		Manifest   bool   `json:"is_manifest_list"`
-		Reversion  bool   `json:"reversion"`
-		StartTime  int    `json:"start_ts"`
+		Name     string                 `json:"name"`
+		Manifest bool                   `json:"is_manifest_list"`
+		Others   map[string]interface{} `json:"-"`
```

Comment:
Happy to be corrected, but I don't think `Others` gets populated with any values when unmarshalling so this will always be empty. I think it is clearer to just have Name and Manifest if that is all we need.